versions.lock

```diff
@@ -102,26 +102,20 @@ com.google.auto.value:auto-value-annotations:1.7.4 (1 constraints: 640a29b9)
 com.google.testing.compile:compile-testing:0.19 (1 constraints: de04f630)
 com.google.truth:truth:1.1 (1 constraints: b71111d7)
 com.helger:profiler:1.1.1 (1 constraints: e21053b8)
-com.netflix.concurrency-limits:concurrency-limits-core:0.2.2 (1 constraints: 7d14fe7b)
 com.netflix.feign:feign-core:8.18.0 (3 constraints: de3f76e0)
 com.netflix.feign:feign-jackson:8.18.0 (1 constraints: c718909e)
 com.palantir.conjure:conjure-core:4.36.0 (1 constraints: 3f05553b)
 com.palantir.conjure.java.api:test-utils:2.38.0 (1 constraints: 3c05463b)
 com.palantir.conjure.java.runtime:conjure-java-annotations:7.66.0 (1 constraints: c918989e)
 com.palantir.conjure.java.runtime:conjure-java-jaxrs-client:7.66.0 (1 constraints: 3f05613b)
 com.palantir.conjure.java.runtime:conjure-java-jersey-server:7.66.0 (1 constraints: 3f05613b)
-com.palantir.conjure.java.runtime:conjure-java-legacy-clients:7.66.0 (3 constraints: c23287a2)
-com.palantir.conjure.java.runtime:conjure-java-retrofit2-client:7.66.0 (1 constraints: 3f05613b)
-com.palantir.conjure.java.runtime:okhttp-clients:7.66.0 (1 constraints: 421ae10c)
-com.palantir.conjure.java.runtime:refresh-utils:7.66.0 (2 constraints: 0a33eabc)
+com.palantir.conjure.java.runtime:conjure-java-legacy-clients:7.66.0 (2 constraints: 071ecd43)
+com.palantir.conjure.java.runtime:refresh-utils:7.66.0 (1 constraints: c918989e)
 com.palantir.safe-logging:preconditions-assertj:3.6.0 (1 constraints: 0b050c36)
 com.palantir.tracing:tracing-jersey:6.18.0 (1 constraints: 47199dbb)
-com.palantir.tracing:tracing-okhttp3:6.18.0 (1 constraints: b814e190)
 com.palantir.websecurity:dropwizard-web-security:1.1.0 (1 constraints: 0405f335)
-com.squareup.okhttp3:okhttp:3.14.1 (3 constraints: fe226150)
+com.squareup.okhttp3:okhttp:3.14.1 (1 constraints: 3b05423b)
```

Comment:
🚀

Reply:
This is likely never going to 0, but its ok because it is a `testImplementation` configuration only.